official/ubuntu_16_04/docker/laas/Dockerfile

```diff
@@ -0,0 +1,64 @@
+MAINTAINER Vincent Berenz <vincentberenz@tuebingen.mpg.de>
+
+# [/BASH IGNORE]
+
+###################################################################################################
+# Install robotpkg dependencies and other dependencies for dynamic graph
+###################################################################################################
+
+#[BASH UPDATE]
+# Add the robotpkg ppa. This contains the LAAS laboratory code base.
+# Typically: pinocchio, dynamic-graph, gepetto-viewer, sot-core, ...
+RUN echo "deb [arch=amd64] http://robotpkg.openrobots.org/wip/packages/debian/pub $(lsb_release -sc) robotpkg" > /etc/apt/sources.list.d/ros-latest.list
+RUN echo "deb [arch=amd64] http://robotpkg.openrobots.org/packages/debian/pub $(lsb_release -sc) robotpkg" >> /etc/apt/sources.list.d/ros-latest.list
+RUN curl http://robotpkg.openrobots.org/packages/debian/robotpkg.key | sudo apt-key add -
+
+# Install the packages
+
+RUN apt-get -y update --fix-missing
+
+RUN apt-get install -y robotpkg-dynamic-graph-v3      # The dynamic graph
+RUN apt-get install -y robotpkg-py27-dynamic-graph-v3 # The dynamic graph python bindings
+RUN apt-get install -y robotpkg-tsid                  # Andrea Delprete Task Space Inverse Dynamics
+RUN apt-get install -y robotpkg-gepetto-viewer        # LAAS 3D robot viewer
+
+RUN apt-get install -y robotpkg-pinocchio             # Eigen based rigid body dynamics library
+RUN apt-get install -y robotpkg-hpp-fcl               # collision detection for pinocchio
+RUN apt-get install -y robotpkg-libccd                # not sure
+RUN apt-get install -y robotpkg-octomap               # not sure
+RUN apt-get install -y robotpkg-parametric-curves     # Spline and polynomes library
+RUN apt-get install -y robotpkg-simple-humanoid-description # Simple humanoid robot_properties package
+RUN apt-get install -y robotpkg-eigen-quadprog        # QP solver using eigen
+
+RUN apt-get install -y robotpkg-sot-core-v3              # Dynamic Graph Utilities
+RUN apt-get install -y robotpkg-sot-tools-v3             # Dynamic Graph Utilities
+RUN apt-get install -y robotpkg-sot-dynamic-pinocchio-v3 # DG wrapper around pinocchio
+# RUN apt-get install -y robotpkg-sot-torque-control       # Andrea dynamic graph entities
+
+RUN apt-get install -y robotpkg-example-robot-data      # Data for LAAS unnitests.
+
+RUN apt-get install -y robotpkg-py27-eigenpy            # Python bindings
+RUN apt-get install -y robotpkg-py27-pinocchio          # Python bindings
+RUN apt-get install -y robotpkg-py27-parametric-curves  # Python bindings
+RUN apt-get install -y robotpkg-py27-sot-core-v3        # Python bindings
+RUN apt-get install -y robotpkg-py27-quadprog           # Python bindings
+# RUN apt-get install -y robotpkg-py27-sot-torque-control # Python bindings
+RUN apt-get install -y robotpkg-py27-sot-dynamic-pinocchio-v3 # Python bindings
+RUN apt-get install -y robotpkg-py27-qt4-gepetto-viewer-corba # LAAS 3D robot viewer network client/server
```

Comment:
Not relevant for this PR but I was wondering if it might be better to list all packages in one `apt-get install` call rather than having multiple calls.  I would expect that this speeds up the build a bit.  It might also be better for docker as I think each `RUN` creates an additional layer (my knowledge on this is limited, though).

Reply:
It's a balance, the downside of having them all in a single line is that the whole image needs to be rebuild if you add or remove a package. If the dockerfile becomes too big, we will no other choice, though.

Reply:
I actually faced an issue recently when installing robotpkg packages in a Singularity image, that the installation failed for some unclear reason when putting them to together in one command, while it works perfectly when installing them separately... So probably we need to keep it like that anyway.